UI/data/locale/en-US.ini

```diff
@@ -1167,3 +1172,60 @@ ContextBar.MediaControls.PlaylistNext="Next in Playlist"
 ContextBar.MediaControls.PlaylistPrevious="Previous in Playlist"
 ContextBar.MediaControls.MediaProperties="Media Properties"
 ContextBar.MediaControls.BlindSeek="Media Seek Widget"
+
+# YouTube Actions and Auth
+YouTube.Auth.Ok="Authorization completed successfuly.\nYou can now close this page."
+YouTube.Auth.NoCode="Sorry, you have not verified authorization."
+YouTube.Auth.WaitingAuth.Title="YouTube User Authorization"
+YouTube.Auth.WaitingAuth.Text="Please complete authorization in your external browser.<br>If the external browser does not open, follow this link and complete the authorization:<br>%1"
+
+YouTube.Actions.CreateNewEvent="Create a new event"
+YouTube.Actions.Title="Title*"
+YouTube.Actions.MyBroadcast="My Broadcast"
+YouTube.Actions.Description="Description"
+YouTube.Actions.Privacy="Privacy*"
+YouTube.Actions.Privacy.Private="Private"
+YouTube.Actions.Privacy.Public="Public"
+YouTube.Actions.Privacy.Unlisted="Unlisted"
+YouTube.Actions.Category="Category"
+
+YouTube.Actions.MadeForKids="Is this video made for kids?*"
+YouTube.Actions.MadeForKids.Yes="Yes, it's made for kids"
+YouTube.Actions.MadeForKids.No="No, it's not made for kids"
+YouTube.Actions.MadeForKids.Help="<a href='https://support.google.com/youtube/topic/9689353'>(?)</a>"
+YouTube.Actions.AdditionalSettings="Additional settings:"
+YouTube.Actions.Latency="Latency"
+YouTube.Actions.Latency.Normal="Normal"
+YouTube.Actions.Latency.Low="Low"
+YouTube.Actions.Latency.UltraLow="Ultra low"
+YouTube.Actions.EnableAutoStart="Enable Auto-start"
+YouTube.Actions.EnableAutoStop="Enable Auto-stop"
+YouTube.Actions.AutoStartStop.Help="<a href='https://developers.google.com/youtube/v3/live/docs/liveBroadcasts#contentDetails.enableAutoStart'>(?)</a>"
+YouTube.Actions.EnableDVR="Enable DVR"
+YouTube.Actions.360Video="360 video"
+YouTube.Actions.360Video.Help="<a href='https://vr.youtube.com/create/360/'>(?)</a>"
+YouTube.Actions.ScheduleForLater="Schedule for later"
+YouTube.Actions.Create_GoLive="Create && Go Live"
```

Comment:
Personally, this doesn’t feel right.
In no other instance I could find, ˋ&ˋ was used (at least not in the UI subdirectory). Just using ˋandˋ in my mind makes it both more readable and more consistent- at the cost of additional length, of course.